tamr_unify_client/dataset/resource.py

```diff
@@ -247,6 +248,20 @@ def from_geo_features(self, features, geo_attr=None):
             self._features_to_updates(features, record_id, key_attrs, geo_attr)
         )
 
+    def upstream_datasets(self):
+        """The Dataset's upstream datasets.
```

Comment:
This needs a note saying something like since the API returns the URIs of the upstream datasets, the resulting list is DatasetURIs, not actual Datasets.